README.md

```diff
@@ -88,11 +88,20 @@ To use certain services like Google Drive, the client secret details must be pro
 }
 ```
 
+**Note:** During development you may need to edit the `manifest.json` with a [Content Security Policy](https://stackoverflow.com/a/56704432) of `'unsafe-eval'`. **Do not commit this change.**
+
+```json
+{
+    "content_security_policy": "script-src 'self' 'unsafe-eval'; object-src 'self'"
```

Comment:
I'm not sure that this is required - We don't use it for development and haven't seen any issues related with CSP.

Reply:
Personally I couldn't get the "Add Archive" page to work locally to unless I added this. I tried both the development and production builds, neither would work.

Reply:
Right now I'd suggest making a separate issue for this, as it hasn't been an issue for us during development on Chrome or Firefox. It might be best to remove it from this PR and chase it separately I think. If it proves to be an environment issue then I'd be more than happy having it added to the docs 😊

Reply:
Sure, I can get rid of this. Just found it was useful in case people get to experience something like this.